src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,6 @@
-const numberToReversedDigits = (number) => {};
-
+const numberToReversedDigits = (number) => {
+	const revNum = number.toString().split("").reverse();
+	const revArray = revNum.map((str) => Number(str));
+	return revArray;
```

Comment:
No need to create the extra variable here, could just return the result of the map function:
```
	return revNum.map((str) => Number(str));
```